property.ts

```diff
@@ -94,10 +129,8 @@ export const checkProperties = (
 
   const simnetAddresses = Array.from(simnetAccounts.values());
 
-  const testContractId = rendezvousList[0];
-
   const testFunctions = getFunctionsListForContract(
-    testContractsTestFunctions,
+    enrichedTestFunctionsInterfacesMap,
```

Comment:
Adding this as a note for us to consider alternatives, otherwise "enriched" it is.